### PR TITLE
[493] Pull image from private Github registry

### PR DESCRIPTION
**Pull request by saliceti**
## What
It is required for some services to keep the package private temporarily. Kubernetes requires a Github user and token to access it.

## How to review
Deployed successfully on `calculate-teacher-pay`

### CODE REVIEW

File: aks/application/variables.tf [hunk 1]
```diff
@@ -92,3 +92,13 @@ variable "probe_command" {
   description = "Command for the liveness and startup probe"
 }
 
+variable "github_username" {
+  type        = string
+  default     = null
+  description = "Github user authorised to access the private registry"
+}
+variable "github_pat" {
```
Comment (thomasleese) [inline aks/application/variables.tf:100]: Could we call this something like like `github_personal_access_token` or `github_token` or `github_password`?

I didn't immediately know what "pat" stood for without needing to read the description.

**Review by thomasleese — COMMENTED** [review]
Looks good, just got one comment about the name of the variable.

File: aks/application/variables.tf (hunk 1 above)
Comment (saliceti) [reply aks/application/variables.tf]: done!

**Review by your — APPROVED** [review]
LGTM

**Review by thomasleese — APPROVED** [review]

**Comment by saliceti** [comment]
[Pull image from private Github package](https://trello.com/c/8iE503j7/493-pull-image-from-private-github-package)

**Pull request merged**